bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -122,70 +128,128 @@ async def autoreview_loop(self) -> None:
             log.info("Running check for users to nominate.")
             await self.reviewer.maybe_review_user()
 
-    @nomination_group.command(
-        name="nominees",
-        aliases=("nominated", "all", "list", "watched"),
-        root_aliases=("nominees",)
+    @tasks.loop(hours=24)
+    async def prune_talentpool(self) -> None:
+        """
+        Prune any inactive users from the talentpool.
+
+        A user is considered inactive if they have sent no messages on the server
+        in the past `DAYS_UNTIL_INACTIVE` days.
+        """
+        log.info("Running task to prune users from talent pool")
+        nominations = await self.api.get_nominations(active=True)
+
+        if not nominations:
+            return
+
+        messages_per_user = await self.api.get_activity(
+            [nomination.user_id for nomination in nominations],
+            days=DAYS_UNTIL_INACTIVE
+        )
+
+        nomination_discussion = self.bot.get_channel(Channels.nomination_discussion)
+        for nomination in nominations:
+            if messages_per_user[nomination.user_id] > 0:
```

Comment:
I think that we should reconsider what "active" means here, having it set to any value > 0 is not quite deterministic, as even 1 message would label them as "active", WDYT ?

Reply:
I see your point.

I guess it's fine to not over engineer this for this particular case. We can always bring it back up if we think it has an impact